pipeline-runner/R/qc-6-integrate_scdata.R

```diff
@@ -235,6 +248,12 @@ run_unisample <- function(scdata, config) {
   scdata <- Seurat::ScaleData(scdata, verbose = FALSE)
   scdata <- Seurat::RunPCA(scdata, npcs = 50, features = Seurat::VariableFeatures(object = scdata), verbose = FALSE)
 
+  if (is.null(npcs)) {
+    message("\nEstimating number of PCs for UMAP\n")
+    npcs <- estimate_npcs(scdata, 0.85, 30)
+    message(sprintf("Number of PCs: %d\n", npcs))
```

Comment:
```suggestion
    message("Number of PCs: ", npcs)
```

`message` does a `paste0` and adds a newline (if you're going for double new line ignore - happy with whatever looks better in the terminal). e.g. try this:

```R
blah <- function() {
    npcs <- 30
    message("Number of PCs: ", npcs)
    message('hello')
}

blah()
```

Reply:
neat. didn't know that. I added the newlines for _pretty printing_, but the messages might be useful in a development scenario, so we could probably do without most of them.

in any case, for the double newline we could use `message("\nNumber of PCs:", npcs, "\n")`